package.json

```diff
@@ -23,10 +23,16 @@
   "homepage": "https://github.com/andela/ah-frontend-sparta#readme",
   "dependencies": {
     "babel-jest": "^24.7.1",
+    "css-loader": "^2.1.1",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.12.1",
+    "postcss": "^7.0.14",
+    "postcss-calc": "^7.0.1",
```

Comment:
It doesn't appear that you are using any of these postcss packages except postcss-loader. Remove extraneous packages

Reply:
@adesege worked on feedback